src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LambdaHandler.java

```diff
@@ -124,8 +124,7 @@ public void checkIndentation() {
      */
     private boolean isNonAcceptableIndent(int astColumnNo, IndentLevel level) {
         return astColumnNo < level.getFirstIndentLevel()
-            || getIndentCheck().isForceStrictCondition()
-               && !level.isAcceptable(astColumnNo);
+            || !level.isAcceptable(astColumnNo, getIndentCheck().isForceStrictCondition());
```

Comment:
Some code changes in Lambda handler, Will share a regression report to check everything is fine for lambda too, seperately.

Reply:
Did regression report, and didn't find any changes.
forceStrictTrue: https://abhishek-kumar09.github.io/Issue3135/
forceStrict False: https://abhishek-kumar09.github.io/IndentaionReport/

Reply:
now IDEA do not like this code - https://teamcity.jetbrains.com/viewLog.html?buildId=3266967&tab=Inspection&buildTypeId=Checkstyle_IdeaInspectionsPullRequest
```
src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation
IndentLevel.java (1)
94: isAcceptable() 'public' method isAcceptable() with 'boolean' parameter 
```

Reply:
It is sad. @Abhishek-kumar09, Please undo the new method.